presto-product-tests/README.md

```diff
@@ -147,6 +147,8 @@ where [profile](#profile) is one of either:
  installation running on a single Docker container and a single node
  installation of kerberized Presto also running on a single Docker container.
  This profile runs Kerberos without impersonation.
+- **singlenode-ldap** - OpenLDAP server installed on top of the environment
+ in [singlenode](#singlenode) profile.
```

Comment:
This is not completely correct. We modify the singlenode profile. I mean for single-node we have `cdh5-hive` and for ldap we have `centos6-java8-oracle-ldap`. I mean we also include certificates which is not a part of single-node.